src/pages/en/core-concepts/astro-pages.md

```diff
@@ -124,6 +124,5 @@ export const get: APIRoute = ({ params, request }) => {
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`. This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
```

Comment:
I’m going to take an executive decision to maintain the whitespace as it was prior to the PR now that we don’t have the separate blockquote here and get this merged. Thanks @tinymachine 🙌 

```suggestion
For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`.

  This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
```